CMakeLists.txt

```diff
@@ -135,6 +135,7 @@ enable_testing()
 add_executable(layer_support_tests
     units/input_buffer_tests.cc
     units/log_scanner_tests.cc
+    units/event_log_tests.cc
```

Comment:
Could you put this unit test file first? It's supposed to be in the alphabetical order.